pkg/plugin/conntrack/_cprog/conntrack.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "vmlinux.h"
+#include "bpf_helpers.h"
+
+// Helper functions to get the current time
+// Ref: https://github.com/cilium/cilium/blob/6186d579ed60f334c7a4daaf81060797b02cc6bd/bpf/lib/time.h
+#define NSEC_PER_SEC	(1000ULL * 1000ULL * 1000UL)
+#define bpf_ktime_get_sec()	\
+	({ __u64 __x = bpf_ktime_get_boot_ns() / NSEC_PER_SEC; __x; })
+# define bpf_mono_now()		bpf_ktime_get_sec()
+
+// These units are in seconds
+
+// Define how long a TCP connection should be kept in the table
+#define CT_CONNECTION_LIFETIME_TCP 360 
+// Define how long a non-TCP connection should be kept in the table
+#define CT_CONNECTION_LIFETIME_NONTCP 60
+// Define how long a TCP connection should be kept alive after receiving the first SYN
+#define CT_SYN_TIMEOUT 60
+// Define the interval at which a packet should be sent to the userspace
+#define CT_REPORT_INTERVAL 30
+// Define the maximum number of connections that can be stored in the conntrack table
+#define CT_MAP_SIZE 262144
```

Comment:
how did we derive this value?

Reply:
![image](https://github.com/user-attachments/assets/0a76ad5c-5ebb-48f0-a1f9-6377368eb71d)
I picked this number initially because it aligns with native conntrack table size, but we can always modify this to something more suitable